net/pfSense-pkg-haproxy-devel/files/usr/local/www/shortcuts/pkg_haproxy.inc

```diff
@@ -24,6 +24,10 @@ global $shortcuts;
 
 $shortcuts['haproxy'] = array();
 $shortcuts['haproxy']['main'] = "/haproxy/haproxy_global.php";
-$shortcuts['haproxy']['log'] = "/status_pkglogs.php?pkg=haproxy-devel";
+if (substr(trim(file_get_contents("/etc/version")),0,3) < 2.5) {
+        $shortcuts['haproxy']['log'] = "/status_pkglogs.php?pkg=haproxy-devel";
+        } else {
+        $shortcuts['haproxy']['log'] = "/status_logs_packages.php?pkg=haproxy-devel";
+        }
```

Comment:
The indent of `} else {` and the closing `}` is too far over, they should be aligned with the `if {`.